src/main/java/org/embulk/output/GcsOutputPlugin.java

```diff
@@ -78,6 +88,26 @@ public TransactionalFileOutput open(TaskSource taskSource, final int taskIndex)
         return new GcsTransactionalFileOutput(task, client, taskIndex);
     }
 
+    private void deleteFiles(PluginTask task)
+    {
+        logger.info("delete files");
```

Comment:
This message isn't clear to me what this plugin is doing.
"Start delete files operation" or something might be better?

Reply:
thanks, I changed this to your suggestions.